RFXtrx/__init__.py

```diff
@@ -723,6 +736,35 @@ def close_connection(self):
         self.transport.close()
         self._thread.join()
 
+    def set_recmodes(self, modenames):
+        """ Sets the device modes (which protocols to decode) """
+        data = bytearray([0x0D, 0x00, 0x00, 0x00, 0x03, 0x00, 0x00,
+                          0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00])
+
+        # Keep the values read during init.
+        data[5] = self._status.device.tranceiver_type
+        data[6] = self._status.device.output_power
+
+        # Build the mode data bytes from the mode names
+        for mode in modenames:
+            byteno, bitno = lowlevel.get_recmode_tuple(mode)
+            if byteno is None:
+                raise ValueError('Unknown mode name '+mode)
+
+            data[7 + byteno] |= 1 << bitno
+
+        self.transport.send(data)
+        return self.transport.receive_blocking()
+
+    def send_start(self):
+        """ Sends the Start RFXtrx transceiver command """
+        self.transport.send(b'\x0D\x00\x00\x03\x07\x00\x00\x00\x00\x00\x00\x00\x00\x00')
```

Comment:
We did not send this before?
So why is this necessary?

Reply:
Do you mean the "send_start" call?
I know that it worked without it but the rfxcom SDK says that it's part of the things to do when starting dialogue with the device. I think we should keep it as it does not interfere with the other commands and it's only used once and the SDK says we should use it anyway.